docs/sources/mimir/get-started/_index.md

```diff
@@ -156,31 +156,30 @@ scrape_configs:
       - targets: ["localhost:9090"]
 ```
 
-## Configure Grafana Agent to write to Grafana Mimir
+## Configure Grafana Alloy to write to Grafana Mimir
 
-Add the following YAML snippet to one of your Agent metrics configurations (`metrics.configs`) in your Agent configuration file, and restart Grafana Agent:
+Use the `prometheus.remote_write` component in Grafana Alloy to send metrics to Grafana Mimir. For example:
 
-```yaml
-remote_write:
-  - url: http://localhost:9009/api/v1/push
 ```
+prometheus.remote_write "LABEL" {
+  endpoint {
+    url = REMOTE_WRITE_URL
 
-The configuration for an Agent that scrapes itself for metrics and writes those metrics to Grafana Mimir looks similar to this:
+    ...
+  }
 
-```yaml
-metrics:
-  wal_directory: /tmp/grafana-agent/wal
-
-  configs:
-    - name: agent
-      scrape_configs:
-        - job_name: agent
-          static_configs:
-            - targets: ["127.0.0.1:12345"]
-      remote_write:
-        - url: http://localhost:9009/api/v1/push
+  ...
+}
 ```
 
+For more information about setting up Alloy, refer to [prometheus.remote_write](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/prometheus.remote_write/).
+
+## Monitor Grafana Mimir with the integration for Grafana Cloud
```

Comment:
@eamonryan confrming that recommending the Grafana Cloud integration for monitoring Mimir is what we want to do.

Reply:
It's certainly one of the things yep! Just an option for users.